tasks/tests/unit/test_flare_cleanup.py

```diff
@@ -0,0 +1,127 @@
+import json
+from unittest.mock import call
+
+from shared.django_apps.core.models import Pull, PullStates
+from shared.django_apps.core.tests.factories import PullFactory, RepositoryFactory
+
+from tasks.flare_cleanup import FlareCleanupTask
+
+
+class TestFlareCleanupTask(object):
+    def test_get_min_seconds_interval_between_executions(self):
+        assert isinstance(
+            FlareCleanupTask.get_min_seconds_interval_between_executions(),
+            int,
+        )
+        assert FlareCleanupTask.get_min_seconds_interval_between_executions() > 17000
+
+    def test_successful_run(self, transactional_db, mocker):
+        mock_logs = mocker.patch("logging.Logger.info")
+        mock_archive_service = mocker.patch(
+            "shared.django_apps.utils.model_utils.ArchiveService"
+        )
```

Comment:
Instead of mocking the `ArchiveService`, you could use the `mock_storage` fixture (I believe that should be hooked up to the shared archive service).

That way, you can actually store files (in memory), and assert that those files are truely being deleted from the storage.